modules/integration_azure-firewall/conf/03-health-status.yaml

```diff
@@ -2,7 +2,7 @@ module: Azure firewall
 name: "Health state"
 
 transformation: true
-aggregation: true
+aggregation: ".max(by=['azure_resource_id', 'azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
```

Comment:
Is `azure_resource_id` necessary since we already have name and rg ?

Reply:
Yes indeed, fixed!